reconcile/ocp_release_ecr_mirror.py

```diff
@@ -0,0 +1,267 @@
+import base64
+import json
+import logging
+import subprocess
+import sys
+import tempfile
+
+from urllib.parse import urlparse
+
+from sretoolbox.container import Image
+
+from utils.oc import OC_Map
+from utils.ocm import OCMMap
+
+from reconcile import queries
+from utils.aws_api import AWSApi
+from reconcile.status import ExitCodes
+
+QONTRACT_INTEGRATION = 'ocp-release-ecr-mirror'
+
+LOG = logging.getLogger(__name__)
+
+
+class OcpReleaseEcrMirrorError(Exception):
+    """
+    Used by the OcpReleaseEcrMirror.
+    """
+
+
+class OcpReleaseEcrMirror:
+    def __init__(self, dry_run, instance):
+        self.dry_run = dry_run
+        self.settings = queries.get_app_interface_settings()
+
+        hive_cluster = instance['hiveCluster']['name']
+        cluster_info = self._get_cluster_info(cluster=hive_cluster)
+
+        # Getting the OCM Client for the hive cluster
+        ocm_map = OCMMap(clusters=[cluster_info],
+                         integration=QONTRACT_INTEGRATION,
+                         settings=self.settings)
+        if not ocm_map.ocm_map:
+            raise OcpReleaseEcrMirrorError(f"Can't create ocm client for "
+                                           f"cluster {hive_cluster}")
+        self.ocm_cli = ocm_map.get(hive_cluster)
+
+        # Getting the OC Client for the hive cluster
+        oc_map = OC_Map(clusters=[cluster_info],
+                        integration=QONTRACT_INTEGRATION,
+                        settings=self.settings)
+        self.oc_cli = oc_map.get(hive_cluster)
+        if not oc_map.oc_map:
```

Comment:
perhaps it would be more readable to check if `ocm_map.get(hive_cluster)` exists?